app/js/lib/foxbox.js

```diff
@@ -425,7 +453,7 @@ export default class Foxbox extends Service {
               .then((state) => service.state = state);
           })
         ).then(() => services);
-      });
+      });*/
```

Comment:
Are we going to have state services in the future? If no, let's remove that bit rather than just commenting out.